reconstructSPI/iterative_refinement/expectation_maximization.py

```diff
@@ -444,8 +493,8 @@ def generate_slices(map_3d_f, xy_plane, rots):
             of projection of rotated map_3d_f.
             Shape (n_rotations, n_pix, n_pix)
         xyz_rotated : arr
-            Rotated xy planes.
-            Shape (n_rotations, 3, n_pix**2)
+            Rotated xy planes, with 3D depth added according to z_offset.
```

Comment:
it seems you've returned this without using it, so that you can use it in `insert_slice`. 

This raises the question about a better design and having a separate xy plane with +/0/- z_offset. 

However, I don't think this z_offset will be needed in `insert_slice`